continuous_integration/recipe/meta.yaml

```
@@ -37,6 +37,7 @@ requirements:
     - prompt-toolkit
     - pygments
     - nest-asyncio
+    - nvtx>=0.2.1
```

Not sure if this has any impact on the recipe itself, just suggesting to match up with the syntax of the other deps:

```suggestion
    - nvtx >=0.2.1
```